nodeup/pkg/model/kubeapiserver.go

```diff
@@ -70,6 +71,19 @@ func (b *KubeAPIServerBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(t)
 	}
 
+	// @check if we are using secure client certificates for kubelet and grab the certificates
+	{
+		if b.UseSecureKubelet() {
```

Comment:
I am a big fan of using `{ ... }` to group blocks of code, but here I think we can use the natural block defined by `if b.UseSecureKubelet()`